admin/javascript/src/silverstripe-backend.js

```diff
@@ -153,7 +153,11 @@ class SilverStripeBackend {
 
       const encodedData = encode(refinedSpec.payloadFormat, data);
 
-      return this[refinedSpec.method](endpointSpec.url, encodedData, headers)
+      const args = refinedSpec.method === 'get'
```

Comment:
Might be worth converting the method name to lowercase like we do in `admin/javascript/src/state/records/actions.js`

For example `refinedSpec.method.toLowerCase()`